sysapps/device_capabilities/device_capabilities_api.js

```diff
@@ -12,8 +12,6 @@ var v8tools = requireNative('v8tools');
 var common = requireNative('sysapps_common');
 common.setupSysAppsCommon(internal, v8tools);
 
-var Promise = requireNative('sysapps_promise').Promise;
```

Comment:
There are other parts of the code using it (like the native filesystem code), so I'm removing its uses one module at a time until I'm able to nuke it from orbit.